solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -31,16 +31,16 @@ contract StandardPoolConverter is
     using SafeMath for uint256;
     using MathEx for *;
 
-    IERC20Token private constant ETH_RESERVE_ADDRESS = IERC20Token(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
-    uint256 private constant MAX_UINT128 = 2**128 - 1;
-    uint256 private constant MAX_UINT112 = 2**112 - 1;
-    uint256 private constant MAX_UINT32 = 2**32 - 1;
-    uint256 private constant AVERAGE_RATE_PERIOD = 10 minutes;
-    uint32 private constant PPM_RESOLUTION = 1000000;
+    IERC20Token internal constant ETH_RESERVE_ADDRESS = IERC20Token(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
+    uint256 internal constant MAX_UINT128 = 2**128 - 1;
+    uint256 internal constant MAX_UINT112 = 2**112 - 1;
+    uint256 internal constant MAX_UINT32 = 2**32 - 1;
+    uint256 internal constant AVERAGE_RATE_PERIOD = 10 minutes;
+    uint32 internal constant PPM_RESOLUTION = 1000000;
 
-    uint256 private __reserveBalances;
-    IERC20Token[] private __reserveTokens;
-    mapping(IERC20Token => uint256) private __reserveIds;
+    uint256 internal __reserveBalances;
```

Comment:
the logic of having 2 balances in the same var shouldn't be exposed to anyone (including child contracts).

Reply:
I've changed all vars from `private` to `internal`. Do you want to change only `__reserveBalances` back to `private`?

Reply:
you only need to change the ones that make sense to change - the storage optimization should be encapsulated within the parent contract.

Reply:
you only need to change the ones that make sense to change - the storage optimization should be encapsulated within the parent contract.

Reply:
Fixed.